File: eTierMeeting_v2/API/Helpers/Params/InventoryParams.cs
using Machine_API.DTO;

namespace Machine_API.Helpers.Params
{
    public class InventoryParams
    {
        public int IdInventory { get; set; }
        public string IdPlno { get; set; }
        public string FromDateTime { get; set; }
        public string ToDateTime { get; set; }
        public List<SearchMachineInventoryDto> listMachineInventory { get; set; }
    }
}

File: eTierMeeting_v2/SPA/src/app/_core/_dtos/inventory-params.ts
import { SearchMachineInventory } from '../_models/search-machine-inventory';

export interface InventoryParams {
    idInventory: number;
    idPlno: string;
    fromDateTime: string;
    toDateTime: string;
    listMachineInventory: SearchMachineInventory
    erorr: number;
    inventoryID: number;
    typeFile: string;
}